baselines/dom.generated.d.ts

```diff
@@ -18266,8 +18266,6 @@ interface PerformanceResourceTiming extends PerformanceEntry {
     readonly requestStart: DOMHighResTimeStamp;
     /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming/responseEnd) */
     readonly responseEnd: DOMHighResTimeStamp;
-    /** [MDN Reference](https://developer.mozilla.org/docs/Web/API/PerformanceResourceTiming/responseStart) */
-    readonly responseStart: DOMHighResTimeStamp;
```

Comment:
Full Safari and firefox support is there.
https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/responseStart#browser_compatibility

Reply:
MDN hasn't been updated yet, but I don't think the BCD PR is right.

Reply:
I don't understand why we're deleting this? `responseStart` has been supported for a long time. It's just what "responseStart" mean changed slightly in Chrome 115, and has been changed back since Chrome 133. See: https://chromestatus.com/feature/5158830722514944. That's why we updated BCD to make a noted of that.

But in terms of typescript, `reponseStart` was always present and was always a `DOMHighResTimeStamp`. So that note shouldn't change anything in typescript.